tests/test_halo_update.py

```diff
@@ -547,3 +548,88 @@ def get_horizontal_dims(dims):
     else:
         raise ValueError(f"no y dimension in {dims}")
     return y_dim, x_dim
+
+
+@pytest.mark.parametrize(
+    "layout, n_points, n_points_update, n_buffer",
+    [((1, 1), 2, "more", 0)],
+    indirect=True,
+)
+def test_halo_updater_stability(
+    zeros_quantity_list,
+    communicator_list,
+    n_points_update,
+    n_points,
+    numpy,
+    subtests,
+    boundary_dict,
+    ranks_per_tile,
+):
+    """
+    Test that that halo_updater is stable both in data exchange and that the
```

Comment:
What does stable mean? Can you replace stable with more explicit words?

Reply:
```
Test that that halo_updater.start()/wait() is consistent through multiple execution.
Test the internal buffers are re-used properly and re-cached properly.
```